crates/wash-cli/src/up/mod.rs

```diff
@@ -272,6 +270,10 @@ pub struct WasmcloudOpts {
     /// If enabled, allows starting additional wasmCloud hosts on this machine
     #[clap(long = "multi-local")]
     pub multi_local: bool,
+
+    /// Defines the Max Execution time (in humantime eg : 2h45min) that the host runtime will execute for
+    #[clap(long = "max-execution-time", alias = "max-time", env = WASMCLOUD_MAX_EXECUTION_TIME_MS, default_value = DEFAULT_MAX_EXECUTION_TIME)]
+    pub max_execution_time: humantime::Duration,
```

Comment:
I'm a little worried about the inconsistency about using humantime here but standard Durations for the other args, since we've aimed to keep the environment variables the same w/ `wash` and `wasmCloud`. It's less fun, but I think a `u64` would suffice here for the execution time.